nav2_rviz_plugins/include/nav2_rviz_plugins/lifecycle_status_indicator.hpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIFECYCLE_STATUS_INDICATOR_HPP_
+#define LIFECYCLE_STATUS_INDICATOR_HPP_
+
+#include <QtWidgets>
+
+#include "rviz_default_plugins/visibility_control.hpp"
+
+namespace nav2_rviz_plugins
+{
+
+class LedStatusIndicator : public QWidget {
```

Comment:
Instead of creating a custom widget, could we do this with a standard widget instead, like a checkbox, or even just a label that changes text color?

Reply:
For sure, we can do that! Maybe I got a bit too excited creating a new widget, I wanted to add a little colour circle changing colour with the changes on the state, and that´s why I thought it would be good to create a new widget.
Maybe a checkbox could do the same, although it might also need some styling to look good.
If you like, I have no problem in changing it to any of those options and adding the necessary changes in nav2_panel instead. It might end up looking better :)